frontend/public/locales/fr/quiz.json

```diff
@@ -91,8 +91,8 @@
     },
     "question-where": {
       "option-canada-ft": "Au Canada",
-      "option-canada-pt-60-or-more": "Au Canada <span>plus</span> de 6 mois par année",
-      "option-canada-pt-less-than-60": "Au Canada <span>moins</span> de 6 mois par année",
+      "option-canada-pt-60-or-more": "Au Canada plus de 6 mois par année",
+      "option-canada-pt-less-than-60": "Au Canada moins de 6 mois par année",
```

Comment:
If you remove the `<span>` here, then you also have to modify the associated lines in `src/components/quiz/questions/QuestionWhere.tsx`.

If you're not comfortable doing that, I would suggest to leave the `<span>` in and ask one of the developers to remove it.

Reply:
got it - will ask Frank

Reply:
👍🏻 In that case, please add the `<span>` elements back in and  Frank can remove them in a future commit, along with the associated lines in `QuestionWhere.tsx`.

Reply:
Did you add the `<span>` back in both places? I still don't see it in the french quiz.json.

> I made the change in my fork - do I need to put in another PR?

No, your PR will update as you update your fork's branch.

Reply:
done - updated in EN and FR